model/src/main/scala/aqua/model/OpModel.scala

```diff
@@ -142,9 +167,15 @@ case class CallServiceModel(serviceId: ValueModel, funcName: String, call: CallM
 }
 
 object CallServiceModel {
-  def apply(serviceId: String, funcName: String, args: List[ValueModel], result: VarModel): CallServiceModel =
+
+  def apply(
+    serviceId: String,
+    funcName: String,
+    args: List[ValueModel],
+    result: VarModel
+  ): CallServiceModel =
     CallServiceModel(
-      LiteralModel(s"\"$serviceId\"", ScalarType.string),
+      LiteralModel.liftScalarString(serviceId),
```

Comment:
scalars cannot be literals. I think there can be some mess with variable names that, for some reason, were quoted